models/bamf_ct_liver_tumor/utils/NNUnetRunnerV2.py

```diff
@@ -0,0 +1,115 @@
+"""
+-------------------------------------------------
+MHub - NNU-Net Runner v2
+       Runner for pre-trained nnunet v2 models. 
+-------------------------------------------------
+
+-------------------------------------------------
+Author: Rahul Soni
+Email:  rahul.soni@bamfhealth.com
+-------------------------------------------------
+"""
+
+
+from typing import List, Optional
+import os, subprocess, shutil
+import SimpleITK as sitk, numpy as np
+from mhubio.core import Module, Instance, InstanceData, DataType, FileType, IO
+
+
+nnunet_dataset_name_regex = r"Dataset[0-9]{3}_[a-zA-Z0-9_]+"
+
+@IO.ConfigInput('in_data', 'nifti:mod=mr', the="input data to run nnunet on")
+@IO.Config('nnunet_dataset', str, None, the='nnunet dataset name')
+@IO.Config('nnunet_config', str, None, the='nnunet model name (2d, 3d_lowres, 3d_fullres, 3d_cascade_fullres)')
+@IO.Config('folds', int, None, the='number of folds to run nnunet on')
+@IO.Config('use_tta', bool, True, the='flag to enable test time augmentation')
+@IO.Config('roi', str, None, the='roi or comma separated list of roi the nnunet segments')
```

Comment:
Since this is a custom implementation of the NNUnetRunner module (which is totally fine but requires less generalization) which of these configurables do actually make sense to be alternatively specified by a potential user? I suggest to keep everything as specific as possible and only allow for customizations where user-adaptions are meaningful and expected.

Reply:
There configurations might not be altered by the users as it is model specific. Would it be better if we combine NNUnetRunnerV2 execution logic into next step PostProcessor.py?

Reply:
@LennyN95 I have formatted NNUnetRunnerV2 to use fewer params. could you please check on this so that same changes can be done across other PRs.

Reply:
Thank you @jithenece, I added a review comment below.